packages/clerk-js/src/ui/foundations/colors.ts

```diff
@@ -37,6 +37,7 @@ export const colors = Object.freeze({
   ...whiteAlpha,
   colorBackground: 'white',
   colorInputBackground: 'white',
+  colorAvatarGradient: [],
```

Comment:
```suggestion
  colorAvatarGradient: ['#6C47FF', '#5BC5EF', '#FBD486', '#BEF972', '#FF8F8F'],
```

Let's move the default avatar colors inside the theme and let `parseAppearance` handle merging the default with the user-provided values.

You're going to need a couple of more changes, writing them in comments as well